source/lmp/fix_dplr.cpp

```diff
@@ -157,6 +192,48 @@ void FixDPLR::init() {
   // 	 << vv[ii][2] << " "
   // 	 << endl;
   // }
+  // check variables
+
+  if (xstr) {
+    xvar = input->variable->find(xstr);
+    if (xvar < 0)
+      error->all(FLERR, "Variable {} for x-field in fix {} does not exist",
+                 xstr, style);
```

Comment:
please add `{` and `}` around the line.
The same issue for all single line `if` statements.

Reply:
@wanghan-iapcm Do we need to add this option: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#insertbraces

Reply:
> @wanghan-iapcm Do we need to add this option: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#insertbraces

Yes please.

Reply:
I submit #2629.

Reply:
UT has been added now.

Reply:
> please add `{` and `}` around the line. The same issue for all single line `if` statements.

Braces added.

Reply:
Document also added.